ocaml/xapi/message_forwarding.ml

```diff
@@ -2065,6 +2065,10 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 			let host = Db.PBD.get_host ~__context ~self:pbd in
 			do_op_on ~local_fn:(Local.VM.import ~url ~sr ~full_restore ~force) ~__context ~host (fun session_id rpc -> Client.VM.import rpc session_id url sr full_restore force)
 
+		let xenprep_start ~__context ~self =
+			info "VM.xenprep_start: VM = '%s'" (vm_uuid ~__context self);
+			Local.VM.xenprep_start ~__context ~self
```

Comment:
It doesn't change the behaviour given that the function at the end of the chain doesn't really do anything.
But I suppose it'll be needed once we put in the real implementation, so we might as well have it here now...